package.json

```diff
@@ -56,6 +55,9 @@
     "web-vitals": "^1.0.1"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.16.7",
+    "@babel/plugin-proposal-private-property-in-object": "^7.16.7",
+    "@open-wc/webpack-import-meta-loader": "^0.4.7",
```

Comment:
Required to handle private properties on classes, see https://github.com/polkadot-js/api/issues/4500